tests/test_host.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from illuminatio.host import (
+    ClusterHost,
+    ConcreteClusterHost,
+    ExternalHost,
+    LocalHost,
+    GenericClusterHost,
+    Host,
+)
+
+
+@pytest.mark.parametrize(
+    "identifier,expected_host",
+    [
+        pytest.param("localhost", LocalHost(), id="LocalHost"),
+        pytest.param(
+            "127.0.0.1", ExternalHost("127.0.0.1"), id="ExternalHost with localhost IP"
+        ),
+        pytest.param(
+            "123.456.789.148",
```

Comment:
Do we accept a wrong IP address?

Reply:
Good point, I'll give the IP Handling a second thought